packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -120,26 +115,11 @@ class WebDevice extends Device {
     bool usesTerminalUi = true,
     bool ipv6 = false,
   }) async {
-    await buildWeb(
-      package.flutterProject,
-      fs.path.relative(mainPath, from: package.flutterProject.directory.path),
-      debuggingOptions.buildInfo,
-    );
-    _package = package;
-    _server = await HttpServer.bind(InternetAddress.loopbackIPv4, 0);
-    _server.listen(_basicAssetServer);
-    printStatus('Serving assets from http:localhost:${_server.port}');
-    await chromeLauncher.launch('http://localhost:${_server.port}');
     return LaunchResult.succeeded(observatoryUri: null);
```

Comment:
This is probably where folks will come looking for the code that starts the app, so if this is not where that code lives, a comment pointing to the code that does it will likely be helpful.